src/components/SliderBox/types/sliderTypes.ts

```diff
@@ -0,0 +1,10 @@
+export interface Slides {
+  name: string;
+  date: string;
+  imageURL: string;
+}
+
+export const enum ArrowDirection {
+  PREVIOUS = 'previous',
+  NEXT = 'next',
+}
```

Comment:
this enum should be moved to constants folder

Reply:
fixed